src/var/mash.jl

```diff
@@ -0,0 +1,57 @@
+# MinHash
+# ========
+#
+# Functions to get MASH distance for MinHash sketches
+#
+# see DOI: 10.1186/s13059-016-0997-x
+#
+# This file is a part of BioJulia.
+# License is MIT: https://github.com/BioJulia/Bio.jl/blob/master/LICENSE.md
+
+function jaccarddistance(sketch1::MinHashSketch, sketch2::MinHashSketch)
```

Comment:
Looks like the way Distances.jl implements this is `1 - sum(min(x, y)) / sum(max(x, y))`, which I think will be less efficient here. The version written here takes advantage of the fact that we know sketches are (1) sets, (2) the same length and (3) sorted. Besides, probably not worth adding another dependency for this one function right?